apps/asap-server/test/controllers/events.test.ts

```diff
@@ -390,6 +390,95 @@ describe('Event controller', () => {
     });
   });
 
+  describe('Past event materials states', () => {
+    const eventId = 'group-id-1';
+    const eventBase = JSON.parse(
+      JSON.stringify(
+        fetchEventsResponse.data.queryEventsContentsWithTotal.items[0],
+      ),
+    );
+
+    test('Should return locked details if details are locked on the CMS', async () => {
```

Comment:
to me `locked` means that whatever the value was set it is locked in place and no longer susceptible to change, whereas the actual state this represents is `permanently unavailable`. I'd definitely prefer to change it `permanentlyUnavailable` or just `unavailable` (still better than locked imo)

Reply:
also I think the test should say that once the features is marked as `locked` (hopefully soon changed to `permanently-unavailable`) the value returned is `null`. I still don't see how `lock` and `null` are related. Making changes to this logic in the feature will be much easier if you follow these two comments, I think.